src/cgame/default/cg_main.c

```diff
@@ -124,7 +125,11 @@ static void Cg_Init(void) {
 	cg_add_weather = cgi.AddCvar("cg_add_weather", "1", CVAR_ARCHIVE,
 	                          "Control the intensity of atmospheric effects.");
 
-	cg_bob = cgi.AddCvar("cg_bob", "1", CVAR_ARCHIVE, "Controls weapon bobbing effect.");
+	cg_autoswitch = cgi.Cvar("autoswitch", "1", CVAR_USER_INFO | CVAR_ARCHIVE,
+				 "The weapon pickup autoswitch method. 0 disables, 1 autoswitches from Blaster only,"
+				 " 2 always switches, 3 switches to new weapons.");
+
+	cg_bob = cgi.Cvar("cg_bob", "1.0", CVAR_ARCHIVE, "Controls weapon bobbing effect.");
```

Comment:
Actually I'm trying to default these to "1" now, instead of "1.0", to save thrashing in updateBindings. Trust me ;)

Reply:
I think it was 1.0 anyway, but this code current doesn't compile because master to the current upstream version of this branch has some functions renamed, namely `gi.Cvar` to `gi.AddCvar`.

Will push updates soon.